test/junit/scala/tools/nsc/backend/jvm/opt/InlineWarningTest.scala

```diff
@@ -141,7 +141,7 @@ class InlineWarningTest extends ClearAfterClass {
 
     val warn =
       """M::f()I is annotated @inline but could not be inlined:
-        |The callee M::f()I contains the instruction INVOKESPECIAL M.nested$1 ()I
+        |The callee M::f()I contains the instruction INVOKESTATIC M.nested$1 ()I
```

Comment:
a nice thing about static methods is that we can in principle make them public for the purpose of the optimizer and don't need to name-mangle